.gitignore

```diff
@@ -23,3 +23,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+src/App.js
```

Comment:
Why you have added src/App.js inside .gitignore we want to add the changes made in App.js on Github right?

Reply:
yes for sure! Might because of committing too many files at once. Should not happen I agree.